data/src/config/channel.rs

```diff
@@ -1,20 +1,14 @@
-use serde::{Deserialize, Serialize};
+use serde::Deserialize;
 
-#[derive(Debug, Clone, Default, Deserialize, Serialize)]
+use crate::channel::Position;
+
+#[derive(Debug, Clone, Default, Deserialize)]
 pub struct Channel {
     pub users: Users,
 }
-
-#[derive(Debug, Clone, Copy, Default, Deserialize, Serialize)]
-pub enum Position {
-    Left,
-    #[default]
-    Right,
-}
-
-#[derive(Debug, Clone, Copy, Deserialize, Serialize)]
+#[derive(Debug, Clone, Copy, Deserialize)]
 pub struct Users {
-    pub visible: bool,
+    pub(crate) visible: bool,
```

Comment:
Since this drives the default value of a Setting, we don't allow the app to access it since it should be accessed from the settings